src/communication/azext_communication/__init__.py

```diff
@@ -47,4 +49,26 @@ def load_arguments(self, command):
             pass
 
 
+class CommunicationCommandGroup(AzCommandGroup):
+
+    def communication_custom_command(self, name, method_name, **kwargs):
+        command_name = self.custom_command(name, method_name, **kwargs)
+        self._register_data_plane_account_arguments(command_name)
+
+    def _register_data_plane_account_arguments(self, command_name):
+        """ Add parameters required to create a communication client """
+        from .manual._validators import validate_client_parameters
+
+        command = self.command_loader.command_table.get(command_name, None)
+
+        if not command:
+            return
+
+        group_name = 'communication'
+        command.add_argument('connection_string', '--connection-string', required=False, default=None,
```

Comment:
Does this add --connection-string as an argument to the CLI? If so, shouldn't --connection-string be in all the examples in the help file?

Reply:
For end user, --connection-string is not required to mention in the command. It added internally either through config file or an environment variable when any commands get executed. I believe, we don't need to mention.